clfd/tests/test_pfd_processing.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+import os
+import tempfile
+
+import numpy
+from clfd import DataCube, profile_mask, featurize, time_phase_mask
+from clfd.interfaces import PfdInterface
+
+from utils import get_example_data_path
+
+try:
+    from ptypes.presto import PTypePFD
+    HAS_PTYPES = True
+except ImportError:
+    HAS_PTYPES = False
+
+class TestPfdProcessing(unittest.TestCase):
+
+    """
+    Test PFD interface.
+    """
+
+    def setUp(self):
+        self.pfd_data_fname = os.path.join(get_example_data_path(),
+                                           "pfd_example.pfd")
+        self.nchan = 4096
+
+        # frequency offset (i.e. channel width),
+        # negative because the first channel has
+        # the highest frequency.
+
```

Comment:
But clearly the channel offset is positive for these data. I suppose this comment is a leftover from copy pasting, and should be deleted.

Reply:
Yup. Deleted :grin:.